haproxy-2.9.yaml

```
@@ -32,10 +24,12 @@ environment:
       - pcre2-dev
 
 pipeline:
-  - uses: fetch
+  - uses: git-checkout
     with:
-      uri: https://www.haproxy.org/download/${{vars.mangled-package-version}}/src/haproxy-${{package.version}}.tar.gz
-      expected-sha256: d1a0a56f008a8d2f007bc0c37df6b2952520d1f4dde33b8d3802710e5158c131
+      repository: https://git.haproxy.org/git/haproxy-2.9.git/
```

Officially listed mirror of ha-proxy on GitHub 
```suggestion
      repository: https://github.com/haproxy/haproxy
```

---

I noticed that! But if you look at the tags in that github repo, they don't seem to follow the same version scheme as the repo above. Although, it does appear to be the official development repository and gets updates more often.

---

github only has develpment tags (rc and final release) but it doesn't have stable maintainance branches or tags which are only on haproxy.org. I.e. the point releases we want to package.

Also this packages 2.9 without a package version subsitituion. I was hoping for a var transform to be included in the uri such that 2.9 is automatically updated.